src/Fixer/Whitespace/BlankLineBeforeStatementFixer.php

```diff
@@ -300,16 +300,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
      */
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
-        $allowed = self::$tokenMap;
-        $allowed['yield_from'] = true; // TODO remove this when update to PHP7.0
-        ksort($allowed);
```

Comment:
maybe we should still sort the keys we are passing to `AllowedValueSubset` ?

Reply:
the token map is already sorted so no need to sort it again I think?

Reply:
only by coincidence, right? nothing is ensuring anymore that values we gonna show as possible options are sorted
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5838/files#diff-e62493a3eac53bf195d7e4e5ba28f97cafe0c59da890c623c2b52082008a2fa9R41
(eg, we can add new value to that structure at end of it and not in alpha order)

Reply:
if I add a utest for that would be enough or are there more thing here that need fixing or should we drop the PR?

Reply:
>only by coincidence, right? 
no, I typed it in that order